quence.lua

```diff
@@ -82,32 +85,31 @@ local steps_copy = {}
 local rests_copy = {}
 local clk = beatclock.new()
 local reverse_play = {}
+local outnum = {}
+local oo = ' '
```

Comment:
Could we have a more descriptive name here?

Reply:
agreed. will fix.